x-pack/plugins/security_solution/public/entity_analytics/translations.ts

```diff
@@ -59,7 +59,7 @@ export const RISK_SCORE_MODULE_STATUS_OFF = i18n.translate(
 export const ENTITY_RISK_SCORING = i18n.translate(
   'xpack.securitySolution.riskScore.riskScorePreview.entityRiskScoring',
   {
-    defaultMessage: 'Entity risk scoring',
+    defaultMessage: 'How is the risk score calculated',
```

Comment:
Is a question mark `?` at the end missing?

Reply:
done.

Reply:
@semd 
This was the issue, I updated the incorrect constant when addressing your [previous comment](https://github.com/elastic/kibana/pull/171089#discussion_r1397115543)
I've corrected it